src/main/types/Errors.ts

```diff
@@ -36,6 +36,7 @@ enum SIOPErrors {
   NO_SELFISSUED_ISS = 'The Response Token Issuer Claim (iss) MUST be https://self-isued.me',
   NO_SUB_TYPE = 'No or empty sub_type found in JWT payload',
   REGISTRATION_NOT_SET = 'Registration metadata not set.',
+  REQUEST_CLAIMS_CANT_SEND_PRESENTATION_DEFINITION_BY_RE_AND_VAL = "Request claims can't have both 'presentation_definition' and 'presentation_definition_uri'",
```

Comment:
RE needs to be REF. Probably something like REQUEST_CLAIMS_PRESENTATION_DEFINITION_BY_REF_AND_VALUE_NON_EXCLUSIVE

Reply:
changed